lib/generators/doorkeeper/templates/initializer.rb

```diff
@@ -108,6 +108,15 @@
   # default_scopes  :public
   # optional_scopes :write, :update
 
+  # Define scopes_by_grant_type to restrict only certain scopes for grant_type
```

Comment:
Maybe it would be better to clarify that in case developer is using `scopes_by_grant_type` - other definitions of the scopes (optional and default) wouldn't be used? WDYT?

Reply:
What if the developer wants to use `scopes_by_grant_type` option but he doesn't want to limit scopes for all the grant_types.
I think it would be better to keep this `scopes_by_grant_type` option separate as It gives more flexibility to the developer.

Reply:
Sorry, I just misunderstood your solution. It's OK, thanks for clarification.